client/js/screens/SignIn/SignIn.js

```diff
@@ -0,0 +1,154 @@
+import React, {useState} from 'react';
+import SafeAreaView from 'react-native-safe-area-view';
+import {
+  ScrollView,
+  Image,
+  ImageBackground,
+  View,
+  Text,
+  TouchableOpacity,
+} from 'react-native';
+import {Form, FormSpy, Field} from 'react-final-form';
+import TitleText from '../../components/TitleText';
+import ButtonDefault from '../../components/ButtonDefault';
+import InputDefaultField from '../../components/InputDefaultField';
+import InputDefaultCheckBox from '../../components/InputDefaultCheckBox';
+import {THEME} from '../../config';
+import styles from './styles';
+
+const SignIn = props => {
+  const [isErrors, setErrors] = useState(null);
+  return (
+    <>
+      <ImageBackground
+        style={styles.signInBgImg}
+        source={require('../../assets/images/imgSignIn01.jpg')}
+      />
+      <ScrollView>
+        <SafeAreaView style={THEME.padding.accountScreen}>
+          <TitleText style={styles.title}>Sign In</TitleText>
+          <Form
+            onSubmit={values => {
+              console.log('submit: ', values);
+            }}>
+            {props => {
+              return (
+                <View>
+                  <FormSpy
+                    subscription={{values: true}}
+                    onChange={({values}) => {
+                      setErrors(null);
+                      if (!values.email) {
+                        setErrors({
+                          ...isErrors,
+                          email: 'Please enter the email',
+                        });
+                      }
+                      if (!values.password) {
+                        setErrors({
+                          ...isErrors,
+                          password: 'Please enter the password',
+                        });
+                      }
+                      // console.log(values, isErrors);
```

Comment:
Please remove all the console logs from your file.

Reply:
Got it